examples/compose/plot_column_transformer.py

```diff
@@ -39,95 +33,130 @@
 from sklearn.compose import ColumnTransformer
 from sklearn.svm import LinearSVC
 
+##############################################################################
```

Comment:
Hi @lucyleeow there is a discussion about changing this syntax starting from 0.23 (#17068). Maybe, keep an eye on it and be ready to change this one if it is merged. Thanks!

Reply:
No problem, easy change!